.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.16
+      - image: circleci/golang:1.17
```

Comment:
Is this the only version supported in Collector and Go SDK? Wondering if this is a considered a breakin change

Reply:
Generally, most repositories seem to support the latest 2 versions of go, which would be 1.17 and 1.18 for us now.